addons/web/static/lib/tempusdominus/tempusdominus.js

```diff
@@ -563,10 +563,10 @@ var DateTimePicker = function ($, moment) {
         DateTimePicker.prototype._notifyEvent = function _notifyEvent(e) {
             // /!\ ODOO FIX: these next conditions have been modified by odoo
             // FIXME should write a test about the tricky case this handles
-            if (!e.date && !e.oldDate) {
-                return;
-            }
             if (e.type === DateTimePicker.Event.CHANGE) {
+                if (!e.date && !e.oldDate) {
+                    return;
+                }
```

Comment:
i honnestly don't know about this... I think that @qsm-odoo input would be useful

Reply:
@aab-odoo Well, this part was about restoring https://github.com/odoo/odoo/commit/ab42507d460125c5ee81bb6b4b859146494339ac with https://github.com/odoo/odoo/commit/43d2db9439f321e356a62de986d59ba43b31436b
But as you can see, the library changed its original condition meanwhile and @kebeclibre restores the old library condition by doing this. I suppose it is ok.